packages/preset-wind/src/index.ts

```diff
@@ -6,22 +6,37 @@ import { rules } from './rules'
 import { shortcuts } from './shortcuts'
 import { theme } from './theme'
 import { variants } from './variants'
+import { postprocessors } from './postprocessors/default'
 
 export { colors, preflights } from '@unocss/preset-mini'
 export type { Theme } from '@unocss/preset-mini'
 
 export { rules, shortcuts, theme, variants }
 
-export interface PresetWindOptions extends PresetMiniOptions { }
+export interface PresetWindOptions extends PresetMiniOptions {
+  /**
+   * The important option lets you control whether UnoCSS’s utilities should be marked with `!important`.
+   *
+   * This can be really useful when using UnoCSS with existing CSS that has high specificity selectors.
+   *
+   * You can also set `important` to a selector like `#app` instead, which will generate `#app .m-1 { ... }`
+   *
+   * @default false
+   */
+  important?: boolean | string
+}
 
 export const presetWind = definePreset((options: PresetWindOptions = {}) => {
+  options.important = options.important ?? false
+
   return {
     ...presetMini(options),
     name: '@unocss/preset-wind',
     theme,
     rules,
     shortcuts,
     variants: variants(options),
+    postprocess: postprocessors(options),
```

Comment:
I feel it would be better to update the variant directly instead of relying on postprocess, which will affect all `!important` even if it's the user's explicit rule that does not come from the important variant.

Reply:
I don't get your point. if you mean it's better to change `important:xxx` variant, then it's another story. The purpose of this PR is as I described in https://github.com/unocss/unocss/issues/3474, which essentially applies to all rules (and that's what Tailwind's `important` global option does too). It will help users fight against those UI library's styles with higher specificity, globally.

Overall, I don't think changing the `important:xxx` variant is a good idea since essentially user will sometimes need to write the real `!important`. Changing its behavior makes no sense.

For your partially opt-in idea, I have a better (at least from my perspective) solution described in https://github.com/unocss/unocss/issues/3474#issuecomment-1873334354, please welcome to have a look and leave your comment.

Reply:
Ok, I see. I thought it was the configuration for the important variant, but actually not. Making sense then, I think we should mention that this would affect all rules.

Reply:
It's tailwind's functionality so I think it makes sense to include it in preset-wind. But it's up to you anyway.

If you want to proceed with that, I'll add a warning block to the doc. Otherwise you can close this and I don't mind to put it into my own preset for my own usages.

By the way what do you think about my `ancestor-xxx:` variant idea https://github.com/unocss/unocss/issues/3474#issuecomment-1873334354? it could be a potential solution for this.

Reply:
I've updated the doc